gulpfile.js

```diff
@@ -1,16 +1,37 @@
 'use strict';
 
+var child_process = require('child_process');
+
 var gulp = require('gulp');
-var beautify = require('gulp-jsbeautifier');
-var jshint = require('gulp-jshint');
-var mocha = require('gulp-mocha');
+
+var models = require('./models');
+var config = require('./lib/config');
 
 var js = ['**/*.js', '!node_modules/**/*.js'];
 var json = ['**/*.json', '!node_modules/**/*.json', '!tests/fixtures/**/*.json'];
 
+gulp.task('run', ['build'], function() {
+    // Create database tables (if needed) and start the server
+    models.sequelize.sync().then(function() {
```

Comment:
Bleh, it looks like `db:migrate` needs a standard Sequelize configuration file [like this](https://gist.github.com/openjck/bdbe892ee2ee44444e25) to exist. Have you discovered a way around that @groovecoder?